requirement-generator/checkers/colleges/ag.ts

```diff
@@ -0,0 +1,122 @@
+import { Course } from '../../types';
+import { courseMatchesCodeOptions } from '../checkers-common';
+
```

Comment:
Not gonna review every single requirement for accuracy as I believe they follow the same logic from reqs.json. Will iron out once the process is set up.